docs/alternator/compatibility.md

```diff
@@ -144,6 +144,35 @@ This monitoring stack is different from DynamoDB's offering - but Scylla's
 is significantly more powerful and gives the user better insights on
 the internals of the database and its performance.
 
+## Time To Live (TTL)
+
+Like in DynamoDB, Alternator items which are set to expire at a certain
+time will not disappear exactly at that time, but only after some delay.
+DynamoDB guarantees that the expiration delay will be less than 48 hours
+(though for small tables the delay is often much shorter).
+
+In Alternator, the expiration delay is configurable - it  can it can be be set
+with the `--alternator-ttl-period-in-seconds` configuration option.
+The default is 24 hours.
+
+ <!--- REMOVE IN FUTURE VERSIONS - Remove the note below in version 5.2. -->
+---
+
+**NOTE**
+
+In ScyllaDB versions 5.0 and earlier, DynamoDB's TTL feature 
+is experimental and needs to be enabled explicitly with the
+`--experimental-features=alternator-ttl` configuration option.
+See [Enabling Experimental Features](#yaml_enabling_experimental_features) for details.
```

Comment:
I thought the idea is that our documentation is up-to-date for the current version. I don't think we need this NOTE about older versions. There are a lot of things that are mentioned in this document - and things dropped from it - which were different in older versions, and we never kept those explanations.

Reply:
I absolutely agree with you. We're going to take this approach starting with version 5.2. I'm going to remove that note; that's why I added the comment "<!--- REMOVE IN FUTURE VERSIONS - Remove the note below in version 5.2. -->".

I'd leave it in version 5.1 because it will be the earliest version on the version drop-down and the only way to find information about previous releases.
Let's say that somebody has version 4.6 - they won't be able to select such a version to learn about the limitation. The information about limitations will only be left in 5.1.

Reply:
I think it's a mistake to add this even temporarily. If you look of the history of this file, there are a lot of things that used to be in this compatibility.md file, but we removed (or changed) them one by one as we added more features. So why single out this specific feature? For example, we had a problem with reverse queries, which we fixed in 5.0 and compatibility.md used to warn about it, and no longer does. Users who use 4.6 will not know that this feature doesn't exist there. And this is just one example. The whole history of this file is features we didn't have and gradually added in different versions.

Reply:
What I meant was that if the feature is not there, that's OK; no need to mention it. But if it's experimental and needs to be enabled with an option, how else would the user know that such an option exists for versions prior to 5.1 if the option is not mentioned anywhere in the docs?
But I see your point and it's convincing to me, so I'll remove that note.

Reply:
The example I gave was of a feature (reverse queries) that was broken in 4.6 and should be avoided, and yet when we fixed it in 5.0, we just removed that text. We didn't leave behind a message saying "Before version 5.0, you need to avoid reverse queries", although we could. The ideas was that somebody who is looking for compatibility.md as it was for version 4.6 should look for it in branch-4.6 in the repository - not in master.